x-pack/plugins/serverless_search/public/plugin.ts

```diff
@@ -69,10 +69,11 @@ export class ServerlessSearchPlugin
 
   public start(
     core: CoreStart,
-    { serverless, management }: ServerlessSearchPluginStartDependencies
+    { serverless, management, observabilityShared }: ServerlessSearchPluginStartDependencies
   ): ServerlessSearchPluginStart {
     serverless.setProjectHome('/app/elasticsearch');
     serverless.setSideNavComponent(createComponent(core, { serverless }));
+    observabilityShared.setIsSidebarEnabled(false);
```

Comment:
Out of curiosity,  Why do we have to this to false?

Reply:
In Serverless we don't want the solution-specific sidebars to show up, because Serverless has its own, always-on global sidebar.